src/main/java/net/conjur/api/Endpoints.java

```diff
@@ -32,7 +32,8 @@ public URI getSecretsUri() {
     public static Endpoints fromSystemProperties(){
 
         String account = System.getProperty(ACCOUNT_PROPERTY_NAME);
-
+        if (account == null)
+            System.out.println("ERROR: " + ACCOUNT_PROPERTY_NAME + " env variable is undefined");
```

Comment:
I'm a little concerned with writing the error message to the standard output. The SDK currently doesn't do that anywhere else so it's not an expected behavior by the user. If we add messages, I think we should use a logger class that can be toggled on and off, based on configuration.

Reply:
I also suggest rephrasing the error message:

```
Error: environment variable [" + ACCOUNT_PROPERTY_NAME + "] is undefined
```
Also, what will be the result of this error? According to the code, we continue regardless if this error happens or not. I would add to the message another sentence that explains what would happen as a result, or what should the user do.

Reply:
done